build/common.ps1

```diff
@@ -20,6 +20,8 @@ $DotNetExe = Join-Path $CLIRoot 'dotnet.exe'
 $NuGetExe = Join-Path $NuGetClientRoot '.nuget\nuget.exe'
 $XunitConsole = Join-Path $NuGetClientRoot 'packages\xunit.runner.console.2.1.0\tools\xunit.console.exe'
 $ILMerge = Join-Path $NuGetClientRoot 'packages\ILMerge.2.14.1208\tools\ILMerge.exe'
+$VstsPackageFeedUrl = 'https://devdiv.pkgs.visualstudio.com/_packaging/ApexTestFramework/nuget/v3/index.json'
```

Comment:
I meant assume this will be set as an environment variable, we don’t want to disclose private feeds so please force push